local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -63,12 +67,36 @@ android {
             abiFilters "armeabi-v7a", "x86"
         }
     }
+    splits {
+        abi {
+            enable true
+            reset()
+            universalApk true
+            // Uncomment the line below to create three separate APKs instead of one:
+            //   - A universal APK that works on all devices
+            //   - An APK that only works on ARM devices
+            //   - An APK that only works on x86 devices
+            // The advantage is the size of the APK is reduced by about 4MB
+            // (8MB -> 4MB for a basic app). You just need to upload all these APKs to the
+            // Play Store.
+            // include "armeabi-v7a", 'x86'
+        }
+    }
     buildTypes {
         release {
             minifyEnabled false  // Set this to true to enable Proguard
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
     }
+    // variants are e.g. debug, release
+    applicationVariants.all { variant ->
+        variant.outputs.each { output ->
+            if (output.getFilter(OutputFile.ABI) != null) {  // null for the universal-debug, universal-release variants
+                output.versionCodeOverride =
+                        versionCodes.get(output.getFilter(OutputFile.ABI)) * 1048576 + defaultConfig.versionCode
```

Comment:
Also explained here: http://tools.android.com/tech-docs/new-build-system/user-guide/apk-splits

Very good point about the comment, will add it.